tests/python/relay/test_op_level3.py

```diff
@@ -1241,13 +1238,16 @@ def test_scatter_add(self, target, dev, ref_data, dshape, ishape, axis, dtype):
     ],
 )
 def test_gather(target, dev, executor_kind, data, axis, indices, ref_res):
-    def verify_gather(data, axis, indices, ref_res):
+    def verify_gather(data, axis, indices, ref_res, check_negative=False):
```

Comment:
Seems like check_negative is never set to anything other than False with the current tests. remove?

Reply:
Nope, one of the tests is supposed to check negatives, thanks for the catch